Sources/Token+URL.swift

```diff
@@ -117,7 +117,7 @@ private func urlForToken(name: String, issuer: String, factor: Generator.Factor,
     return url
 }
 
-private func tokenFromURL(_ url: URL, secret externalSecret: Data? = nil) -> Token? {
+private func token(from url: URL, secret externalSecret: Data? = nil) -> Token? {
```

Comment:
Ignoring these warnings, as the represent issues that already exist and aren't being exacerbated by this PR.